.github/workflows/pr.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33070&commitId=110c6a858151ed29fed4e9e80b332d8f1934c760&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Right top edge.chromium.png|1127| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|
|Callout.Root.chromium.png|2309| Changed|

</details>


  <details><summary><b>Keytip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Keytip.Offset.chromium.png|121| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>